src/webid-oidc.js

```diff
@@ -61,13 +61,20 @@ export const currentSession = async (
   }
 }
 
-export const logout = (storage: AsyncStorage): Promise<void> =>
+export const logout = (storage: AsyncStorage, idp: string): Promise<void> =>
   getStoredRp(storage)
     .then(rp => (rp ? rp.logout() : undefined))
     .catch(err => {
       console.warn('Error logging out of the WebID-OIDC session')
       console.error(err)
     })
+    .then(x => {
+      fetch(idp + '/logout', { method: 'GET', credentials: 'include'})
```

Comment:
This should move up, and there should only be one `catch` block.

(Also, unsure why we're using the `then/catch` syntax here instead of `async/await`.)

Reply:
Also, if this is a state-changing operation, this cannot be a `GET`!

Reply:
Oh, that's a bug in NSS: https://github.com/solid/node-solid-server/issues/744

Reply:
@RubenVerborgh I have fixed code GET was replaced with POST and call was moved up.